test/Quake/dependency-1.qke

```diff
@@ -6,11 +6,11 @@
 // the terms of the Apache License 2.0 which accompanies this distribution.   //
 // ========================================================================== //
 
-// RUN: cudaq-opt --assign-ids --dep-analysis %s | FileCheck %s
+// RUN: cudaq-opt --add-wireset --assign-wire-indices %s | FileCheck %s
```

Comment:
Are these tests passing for you? It looks like they fail for me:
```
Failed Tests (6):
  CUDAQ :: Quake/dependency-1.qke
  CUDAQ :: Quake/dependency-2.qke
  CUDAQ :: Quake/dependency-3.qke
  CUDAQ :: Quake/dependency-4.qke
  CUDAQ :: Quake/dependency-classical.qke
```

Reply:
Those failures look familiar. I thought there was a patch to fix them, but maybe I'm recalling the wrong thing?